### ISSUE TITLE
Script to find missing money

### ISSUE THREAD

**Issue by scientes** [provinzio/CoinTaxman#13]
related to #11 
Maybe a small script which runs before the main one would be nice for these use cases:
if the data for buying or selling is unknown for certain coins (due to the money coming from a deposit or withdrawal or something like that)
print the coins where editing or additional data is needed first before running the script.

Because current behavior of failing when no money is present (due to being from deposit) is suboptimal in my eyes. 

sry if my explanation is not that good.

**Comment by scientes** [comment]
Do we plan to implement things to document stuff like if i send btc to a friend or pay for a service with btc. because that would be processed as a withdrawhal in the taxgenerator and will just be ignored, but the money in that situation is still gone. or is that irrelevant for price calculation? shouldnt that affect the fifo queue?

**Comment by provinzio** [comment]
I haven't thought about this aspect. That's definitely a problem. We could remind the user of this problem by checking all withdrawals and deposits if they match. If something is missing, the most we can do is raise a warning.

**Comment by wearymanateevedaknotstabooresisting** [comment]
The issue is even more complicated. Think about your stock brokerage account - the broker can only do tax calculations with assets that are known to the very broker. If you do a broker change with a partial portfolio, the broker won't recognize them anymore for calculation. However, the new broker recognizes them with the other assets over there. Additionally, if you like donate a stock, the very stock is gone and is not recognized by any.

I assume that the same procedure is how FIFO works for "Private Veraeusserungsgeschaefte" in Germany.

With crypto an additional layer comes by managing your assets privately, in your wallets. However, there one could assume (that you don't have to do this separated accounting for every single address), that there is one "global private wallet accounting" going on for FIFO.

In summary, this is a work intensive problem to tackle when combining different data sources, like X exchanges and private wallets as well as sells due to consume.
